app/grandchallenge/publications/views.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.contrib.messages.views import SuccessMessageMixin
+from django.views.generic import CreateView, ListView
+from guardian.mixins import LoginRequiredMixin
+
+from grandchallenge.core.filters import FilterMixin
+from grandchallenge.publications.filters import PublicationFilter
+from grandchallenge.publications.forms import PublicationForm
+from grandchallenge.publications.models import Publication
+from grandchallenge.subdomains.utils import reverse
+
+
+class PublicationList(LoginRequiredMixin, FilterMixin, ListView):
+    model = Publication
+    paginate_by = 50
+    filter_class = PublicationFilter
+
+    def get_queryset(self):
+        return super().get_queryset().order_by("-created")

```

Comment:
Probably missing a select related.

Reply:
There is nothing to prefetch from the publication model, unfrotauntely, but I will look into the template tag and how we can minimize queries there ;-)

Reply:
Ah ok I see, I wonder then if a better way is to fetch the publications, then prefetch the related archives/algorithms/reader studies/challenges, add an object permissions checker to those?

Reply:
Actually, I realized that you can prefetch the `algorithm_set`, `archive_set` etc. on the list view. This plus adding a non-database-calling check for existence to the template filter before looping over possible related objects does the trick.

Reply:
> Ah ok I see, I wonder then if a better way is to fetch the publications, then prefetch the related archives/algorithms/reader studies/challenges, add an object permissions checker to those?

I'm not sure I folllow. Isn't that what we do already? We're fetching the publication, then in the template filter function we fetch all related objects (now with as few queries as possible) and then we check the current users permission for each of those objects.